partcad/src/partcad/project.py

```
@@ -1426,6 +1427,60 @@ def render(
     ):
         asyncio.run(self.render_async(sketches, interfaces, parts, assemblies, format, output_dir))
 
+    async def async_convert(
+        self,
+        sketches: Optional[List[str]] = None,
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)